Gemfile

```diff
@@ -8,3 +8,5 @@ group :jekyll_plugins do
 	gem 'jekyll-readme-index'
 	gem 'jekyll-sitemap'
 end
+
+gem "webrick", "~> 1.7"
```

Comment:
Why was this change necessary?

Reply:
Ahh, I see. We need to be careful about changing the build dependencies, because right now our "deployment script" is one volunteer with limited technical knowledge. I've split this into a separate PR and asked them to verify that they can make a build before we merge: https://github.com/pkp/pkp-docs/pull/955

(I didn't need to add `webrick` and the build still passed for that PR. If you test the PR and find you can't build without `webrick` we can re-evaluate.)

Reply:
I can confirm the issue still happening here:
`servlet.rb:3:in `require': cannot load such file -- webrick (LoadError)`

But it just happens when I try to run the `jekyll serve`, the `build` command works fine.

Reply:
I see. Can you remove the changes to `Gemfile` and `Gemfile.lock` from this PR and we can sort that out separately? Our build volunteer is on ruby 2.x and I want to minimize the chance of blocking deployment right now. 

We can get everything up to ruby 3.x at some point in the future when there's more time to focus on the docs hub.

Reply:
Done.